kratos/utilities/volume_inside_voxel_utility.cpp

```diff
@@ -0,0 +1,336 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Ariadna Cortes
+//
+
+//Project includes
+#include "volume_inside_voxel_utility.h"
+
+namespace Kratos {
+
+    double VolumeInsideVoxelUtility::NodesApproximation(const GeometryType& rVoxel) {
```

Comment:
I would remove the tabulation in the cpp. Makes the code more compact

Reply:
Adding to @loumalouomega 's suggestion that some time ago we agreed to don't have indentations within namespaces.

Reply:
i'll take it into account thanks!